src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  if (days < 3) {
+    return days * 40;
```

Comment:
You use `days * 40` in several places, so makes sense to crate a variable for this expression